models/_sector/nft/trades/ethereum/platforms/blur_v2_ethereum_base_trades.sql

```diff
@@ -0,0 +1,110 @@
+{{ config(
+    schema = 'blur_v2_ethereum',
+    tags = ['dunesql'],
+    alias = alias('base_trades'),
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_number','tx_hash','sub_tx_trade_id'],
+    )
+}}
+
+{% set blur_v2_start_date = "cast('2023-07-05' as timestamp)" %}
```

Comment:
as a general standard we prefer:
`{% set blur_v2_start_date = '2023-07-05' %}`
with
`WHERE evt_block_time >= TIMESTAMP {{blur_v2_start_date}}`

Nothing wrong with your approach, just trying to use some standards across the repo.

Reply:
I still have to go through a lot of the marketplaces to change this (as they were written before we decided on a standard), so we can leave this as is for now.

Reply:
I used that method because that was the only one that seemingly passed tests, I switched back to your method and now tests are failing again, any clues why?